src/go/rdctl/cmd/snapshotList.go

```diff
@@ -0,0 +1,64 @@
+package cmd
+
+import (
+	"fmt"
+	"os"
+	"sort"
+	"text/tabwriter"
+	"time"
+
+	"github.com/rancher-sandbox/rancher-desktop/src/go/rdctl/pkg/paths"
+	"github.com/rancher-sandbox/rancher-desktop/src/go/rdctl/pkg/snapshot"
+	"github.com/spf13/cobra"
+)
+
+// Like []snapshot.Snapshot but sortable by date created.
+type SortableSnapshots []snapshot.Snapshot
+
+func (snapshots SortableSnapshots) Len() int {
+	return len(snapshots)
+}
+
+func (snapshots SortableSnapshots) Less(i, j int) bool {
+	return snapshots[i].Created.Sub(snapshots[j].Created) < 0
+}
+
+func (snapshots SortableSnapshots) Swap(i, j int) {
+	temp := snapshots[i]
+	snapshots[i] = snapshots[j]
+	snapshots[j] = temp
+}
+
+var snapshotListCmd = &cobra.Command{
+	Use:     "list",
+	Aliases: []string{"ls"},
+	Short:   "List snapshots",
+	RunE: func(cmd *cobra.Command, args []string) error {
```

Comment:
This function is too complex for a `Run(E)` argument. I usually prefer these to do quick command-line checking and then have a helper function do most of the logic.